### PR TITLE
Update calculations for transaction fees

### PR DESCRIPTION
**Pull request by kendallcorner**
Issue #449

### CODE REVIEW

File: payments/management/commands/calculate.py [hunk 1]
```diff
@@ -10,9 +11,10 @@
 class Command(BaseCommand):
 
     def add_arguments(self, parser):
-        parser.add_argument('amount', nargs='+', type=int)
+        parser.add_argument('amount', nargs='+', type=float)
 
     def handle(self, *args, **options):
             amount = options['amount'][0]
+            amount = Decimal(amount)
```
Comment (groovecoder) [inline payments/management/commands/calculate.py]: Could this be combined with the previous line?
```
amount = Decimal(options['amount'][0])
```
?

And/or, we probably need to check that amount had only a single argument before we look at the `0`th element of it?

File: payments/utils.py [hunk 2]
```diff
@@ -21,86 +21,129 @@ def round_penny(amount):
...
+    #
+    #   A = agreed amount
+    #   C = charge = A + 1/2*F
+    #   T = payout transfer = A - 1/2*F
+    #   F = Fees = rp(C*r_sc + f_sc) + rp(A*r_c) + rp(T*r_st)
+    #   r_sc = stripe rate for charges
+    #   f_sc = stripe flat fee for charges
+    #   r_st = stripe rate for payout transfers
+    #   r_c = codesy rate
+    #   rp = round_penny function rounds to the nearest penney
```
Comment (groovecoder) [inline payments/utils.py]: Poetry.

**Comment by jdungan** [comment]
@kendallcorner I'll approve the claim once it goes into production and your code can calculate your payout ... so meta.

**Comment by groovecoder** [comment]
Approving ... my comments aren't worth blocking the merge.

**Comment by kendallcorner** [comment]
Me being the amatuer githubber did not see these comments over the weekend. I can fix Luke's items and add a suggestion from Greg in another PR.

**Pull request merged**